articles/how-to-compile-bitcoin-core-and-run-the-tests.html

```diff
@@ -94,7 +94,7 @@
                       <code>
                         brew install automake berkeley-db4 libtool boost
                         miniupnpc pkg-config python qt libevent qrencode
-                        ccache
+                        ccache sqlite
```

Comment:
Thanks! I think it needs to be sqlite3, though. Can you also place it before ccache? Idem for line 88.

Reply:
For macOS, you may need to check but I wonder if the macOS build doc is wrong and should be sqlite3 as well.

nit: use `git`, and avoid GitHub-specific workflow lock-in as much as possible. Clone the repo locally, pull down the changes, create a named branch, make the changes locally, and push them using git. Use GitHub only for metadata, e.g. review comments.

Reply:
Apologies will do that next time, figured it would of been quicker for such a small change such as this. Thanks for the tip!